mock/mockimpl/svcimpls/mgmtimpl.go

```diff
@@ -0,0 +1,18 @@
+package svcimpls
+
+type mgmtImpl struct {
+}
+
+func (x *mgmtImpl) Register(h *hookHelper) {
```

Comment:
I joined all these, the ordering of the registers matters so it's easier to manage this way. We should possibly look into a way to order hooks for http by most to least precise pathing.

Reply:
Presumably you mean
`/pools/default/buckets/*`
needs to be more important than 
`/pools/default/buckets/*/scopes`
Usually I handle this by having logic such that * only matches a single path, and ** matches multiple depths of path.

Reply:
I think that I mean the inverse, the scopes endpoint needs to matched ahead of the buckets endpoint.